main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/smartatransit/api-gateway/endpoint"
+)
+
+var options struct {
+	JWTSigningSecret string `long:"jwt-signing-secret" env:"JWT_SIGNING_SECRET" required:"true"`
+	ServiceDomain    string `long:"service-domain" env:"SERVICE_DOMAIN" required:"true"`
```

Comment:
whats a service domain?

Reply:
The DNS name of the service itself - this gets added as the `Issuer` of any newly minted anonymous tokens